src/pipecat/services/elevenlabs.py

```diff
@@ -167,34 +180,6 @@ async def set_model(self, model: str):
         await self._disconnect()
         await self._connect()
 
-    async def set_voice(self, voice: str):
-        logger.debug(f"Switching TTS voice to: [{voice}]")
-        self._voice_id = voice
-        await self._disconnect()
-        await self._connect()
-
-    async def set_voice_settings(
```

Comment:
how would this function have been used before? i don't see any usage of it in the codebase and i'm trying to think through if removing this custom-looking handling of the settings breaks things

Reply:
I misunderstood how this could be used. It wasn't accessible before! So, it should be safe to remove.